todo-app/src/App.js

```diff
@@ -1,21 +1,132 @@
 import React from 'react'
 import ToDo from './ToDo.js'
-import {ToDos} from './toDos-DB.js'
+import { ToDos } from './toDos-DB.js'
 
 export default class App extends React.Component {
-  renderToDos(toDos) {
-    return toDos.map((toDo, index) => {
-      return (
-        <ToDo key={index} toDo={toDo} index={index} />
-      )
-
+  constructor(props) {
+    super(props)
+    this.state = {
+      ToDos: ToDos,
+      newToDo: {
+        todoDesc: ""
+      }
+    }
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleNewToDo = this.handleNewToDo.bind(this);
+    this.sendForm = this.sendForm.bind(this);
+    this.deleteToDo = this.deleteToDo.bind(this);
+    this.updateToDo = this.updateToDo.bind(this);
+    this.editToDo = this.editToDo.bind(this);
+  }
```

Comment:
Would be nice to have a new line after each function declaration, just to make it easier to read.